### DISCUSSION TITLE
Floating window on top of everything else.

### DISCUSSION THREAD

**Discussion by SpangeJ** [i3/i3#5803] [Q&A] [answered]
Hello,
I am trying to make a window stay on top even if I enable fullscreen on a video.
So I hit `$mod + Shift + Space` and it becomes floating, but when I click on my (Udemy) video to play in fullscreen it disappears. Is there a way to make the floating window always stay on top/visible no matter what?

**Comment by SpookyB** [comment]
I'm looking for a similar solution. I need to have two windows fullscreened with one floating on top of the other. It can be psuedo-fullscreen if there is a distinction in i3, in fact that might be preferable, as long as it takes up the full display.

**Reply by orestisfl** [reply]
There is an open issue for accepting a maximized command: https://github.com/i3/i3/issues/2107

**Reply by SpookyB** [reply, marked as answer]
I figured out a solution for my problem. To achieve the pseudo fullscreen I had to disable my gaps and bar when switching to the workspace. For my situation that meant doing

`bindsym [binding] [workspace]; exec "polybar-msg cmd hide"; gaps inner current toggle off`

And for every other workspace adding a line that does exec polybar-msg cmd show. This made the window take up the whole screen with the floating window over it. In my case it is a game without a titlebar, but if you application has a titlebar you may have to selectively disable that too to achieve a similar result.
